puppet/services/opendaylight-api.yaml

```diff
@@ -32,6 +36,14 @@ parameters:
   OpenDaylightApiVirtualIP:
     type: string
     default: ''
+  OpendaylightPortBindingController:
+    description: Opendaylight port binding controller
+    type: string
+    default: 'pseudo-agentdb-binding'
```

Comment:
Default will be set to empty string so we will take the default set in networking-odl if unspecified.